### PR TITLE
chore: move storybook to subfolder (outside workspaces)

### PR DESCRIPTION
**Pull request by emmenko**
It's getting more and more frustrating to deal with dependency updates causing conflicts with storybook, as we're still stuck on v5.

So we're moving storybook into a subfolder `docs` with its own package.json and yarn.lock.

The Vercel settings are now as following, to amend the subfolder:

image

### CODE REVIEW

**Comment by emmenko** [comment]
Hmm this is trickier than I thought.

File: package.json [hunk 1]
```diff
@@ -13,21 +13,18 @@
   "keywords": ["javascript", "frontend", "ui-kit", "design-system", "react"],
   "sideEffects": false,
   "scripts": {
-    "postinstall": "preconstruct dev",
+    "postinstall": "preconstruct dev && yarn --cwd docs",
```
Comment (adnasa) [inline package.json:16]: aaah nice!

**Review by adnasa — COMMENTED** [review]
👍  looks good so far!

**Comment by emmenko** [comment]
CI is green, YAY! 🥳

**Review by adnasa — APPROVED** [review]

**Pull request merged**